packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -473,37 +480,54 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
             {children}
           </svg>
         </FocusZone>
+
         {!this.props.hideLegend && (
-          <div ref={(e: HTMLDivElement) => (this.legendContainer = e)} className={this._classNames.legendContainer}>
-            {this.props.legendBars}
-          </div>
+          <React.Suspense fallback={<div>Loading...</div>}>
+            <div ref={(e: HTMLDivElement) => (this.legendContainer = e)} className={this._classNames.legendContainer}>
+              {this.props.legendBars}
+            </div>
+          </React.Suspense>
+        )}
+        {!this.props.hideTooltip && calloutProps!.isCalloutVisible && (
+          <React.Suspense fallback={<div>Loading...</div>}>{callout}</React.Suspense>
```

Comment:
If you initialize the callout as empty jsx then the duplicate if check is not needed.

Reply:
Done